dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides

```diff
@@ -26,13 +29,14 @@ Import-Package:  \
   com.ibm.websphere.ras, \
   com.ibm.websphere.ras.annotation, \
   com.ibm.ws.ffdc, \
+  net.sf.ehcache.*, \
   *
 
 Include-Resource: \
```

Comment:
The include-resources header should only exist in the bnd.bnd file.

Reply:
This was like this with the original bundle. Sure I'll fix it.